content/docs/engine-dev/tools/tiaf/maintaining-tiaf/general-information.md

```diff
@@ -0,0 +1,114 @@
+|Sequence Report File|-report=<filename>|None|Path to where the sequence report file will be written (if this option is not present, no report will be written).|
+|Suite Label Excludes|-labelexcludes=<...>|None|The list of labels that will exclude any tests with any of these labels in their suite.|
+
+The native runtime has the following additional options:
+
+|Option|Flag|Default|Description|
+|:----|:----|:----|:----|
+|Max Concurrency|-maxconcurrency=<number>|Max hardware concurrency|The maximum number of concurrent test targets/shards to be in flight at any given moment (a value of 0 signifies to use the architecture's maximum hardware concurrency).|
+
+# What are the command line options for the TIAF AR scripts?
+
+The following table details the command line options common to both the native and Python AR scripts:
+
+|Option|Flag|Required|Default|Description|
+|:----|:----|:----|:----|:----|
+|Runtime Type|runtime-type=\<native, python>|Yes|None|The runtime TIAF should run tests for.|
+|Runtime Sequence Override|sequence-override=\<tianowrite, seed, tia, regular>|No|None|Manually override the sequence to run with the specified type.|
+|Configuration File|config|Yes|\<tiaf binay build dir>.\<tiaf binary build type>.json|The path to the configuration file for the TIAF runtime.|
+|Source Branch|src-branch|Yes|None|The branch that is being built.|
+|Destination Branch|dst-branch|No|None|For PR builds, the destination branch to be merged to, otherwise empty.|
+|Commit Hash|commit|Yes|None|The commit that is being built.|
+|Build Number|build-number|Yes|None|The build number this run of TIAF corresponds to.|
+|Test Suites|suites|Yes|None|The test suites to select test targets from.|
+|Test Label Excludes|label-excludes|No|None|The CTest labels to exclude test targets from selection if matched.|
+|Test Failure Policy|test-failure-policy|Yes|None|The test failure policy for regular and test impact sequences (ignored when seeding).|
+|Safe Mode|safe-mode|No|None|Run impact analysis tests in safe mode (ignored when seeding).|
+|Test Timeout|test-timeout|No|None|The maximum run time (in seconds) of any test target before being terminated.|
+|Global Timeout|global-timeout|No|None|The maximum run time (in seconds) of the sequence before being terminated.|
+|Test Target Exclusion File|exclude-file|No|None|The path to file containing tests to exclude from this run.|
+|Test Target Output Routing|target-output=\<stdout>|No|None|The test target std/error output routing (if not specified, no test target output will be routed to the console).|
+|S3 Bucket Name|s3-bucket|No|None|The location of S3 bucket to use for persistent storage, otherwise local disk storage will be used.|
+|S3 Bucket Name Top Level Directory|s3-top-level-dir|No|None|The top level directory to use in the S3 bucket.|
+|MARS Index Prefix|mars-index-prefix|No|None|The index prefix to use for MARS, otherwise no data will be transmitted to MARS.|
```

Comment:
suggestion: Might be worth linking to MARS docs if any exist? (ignore if that's not the case)